### PR TITLE
Bump version to 0.6.0

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

**Comment by nehaljwani** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,28 @@
-{% set version = "0.5.4" %}
+{% set name = "isodate" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "2e364a3d5759479cdb2d37cce6b9376ea504db2ff90252a2e5b7cc89cc9ff2d8" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @nehaljwani I really dislike that excessive use of Jinja. Can you revert those changes and keep the recipe simple and readable?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
 package:
-  name: isodate
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: isodate-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/i/isodate/isodate-{{ version }}.tar.gz
-  sha256: 42105c41d037246dc1987e36d96f3752ffd5c0c24834dd12e4fdbe1e79544e31
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: The `fn` can be removed entirely.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -29,6 +33,11 @@ about:
   home: http://cheeseshop.python.org/pypi/isodate
   license: BSD 3-Clause
   summary: 'An ISO 8601 date/time/duration parser and formatter.'
+  description: |
+    This module implements ISO 8601 date, time and duration parsing. The
+    implementation follows ISO8601:2004 standard, and implements only date/time
+    representations mentioned in the standard.
+  dev_url: https://github.com/gweis/isodate
```
Comment (ocefpaf) [inline recipe/meta.yaml:38]: :+1:

**Review by ocefpaf — APPROVED** [review]

**Comment by nehaljwani** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (nehaljwani) [reply recipe/meta.yaml]: :thinking: Removal requires extra effort after `conda skeleton` :sob:

Comment (ocefpaf) [reply recipe/meta.yaml]: Argh, those were added in the skeleton? That is really bad taste IMO.
I always prefer readability vs apparent practicality.

Comment (ocefpaf) [reply recipe/meta.yaml]: Just tried that with conda skeleton and I don't see these extra jinjing:

```yaml
package:
  name: isodate
  version: "0.6.0"

source:
  fn: isodate-0.6.0.tar.gz
  url: https://pypi.python.org/packages/b1/80/fb8c13a4cd38eb5021dc3741a9e588e4d1de88d895c1910c6fc8a08b7a70/isodate-0.6.0.tar.gz
  md5: 0e1203fce27ce65e2d01c5f21c4d428f
#  patches:
   # List any patch files here
   # - fix.patch

# build:
  # noarch: python
  # preserve_egg_dir: True
  # entry_points:
    # Put any entry points (scripts to be generated automatically) here. The
    # syntax is module:function.  For example
    #
    # - isodate = isodate:main
    #
    # Would create an entry point called isodate that calls isodate.main()


  # If this is a new build for the same version, increment the build
  # number. If you do not include this key, it defaults to 0.
  # number: 1

requirements:
  build:
    - python
    - setuptools
    - six

  run:
    - python
    - six

test:
  # Python imports
  imports:
    - isodate
    - isodate.tests

  # commands:
    # You can put test commands to be run here.  Use this to test that the
    # entry points work.


  # You can also put a file called run_test.py in the recipe that will be run
  # at test time.

  # requires:
    # Put any additional test requirements here.  For example
    # - nose

about:
  home: https://github.com/gweis/isodate/
  license: BSD License
  summary: 'An ISO 8601 date/time/duration parser and formatter'
  license_family: BSD

# See
# http://docs.continuum.io/conda/build.html for
# more information about meta.yaml
```

Comment (nehaljwani) [reply recipe/meta.yaml]: Try the latest conda build

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
 
 source:
-  fn: isodate-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/i/isodate/isodate-{{ version }}.tar.gz
-  sha256: 42105c41d037246dc1987e36d96f3752ffd5c0c24834dd12e4fdbe1e79544e31
+  sha256: 2e364a3d5759479cdb2d37cce6b9376ea504db2ff90252a2e5b7cc89cc9ff2d8
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed  --record record.txt
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Can we make this `noarch`?

**Comment by nehaljwani** [comment]
@conda-forge-admin, please add noarch: python

File: recipe/meta.yaml (hunk 4 above)
Comment (nehaljwani) [reply recipe/meta.yaml]: Done by our beloved conda-forge-admin

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml]: conda-forge is still conda-build <3 :unamused:

**Pull request merged**